docs/content/community/bijeenkomsten/20181025-demo-sprint-5.md

```diff
@@ -12,8 +12,39 @@ Amsterdam, 25 oktober 2018
 Slides:
```

Comment:
Bovenaan (ik kan boven regel 12 niets becommentarieren) staat Sprint 4 terwijl het document Sprint 5 in de titel heeft. Wat is het nu?

Reply:
Gefixed